src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Common/SymbolExtensions.cs

```diff
@@ -24,5 +80,64 @@ public static bool NameEquals(this IAssemblySymbol? symbol, string name, bool st
                 return string.Equals(symbol.Name, name, StringComparison.OrdinalIgnoreCase);
```

Comment:
This looks like it should use one of the helper methods identified above
`GetStringComparison`

Reply:
actually, this is an assembly name comparison which is case insensitive AFAIK (maybe depends on the file system, though)

Reply:
Added a comment, though